libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/eh/EHorizonMapper.kt

```diff
@@ -0,0 +1,403 @@
+@file:JvmName("EHorizonMapper")
```

Comment:
This can be removed as all APIs are `internal`.

Reply:
done